_pkgdown.yml

```diff
@@ -28,15 +28,15 @@ navbar:
         - text: Data Flow
           href: articles/blueprint/dataflow.html
         - text: Product Map
-          href: articles/blueprint/products_map.html
+          href: articles/blueprint/product_map.html
         - text: Features
           navbar: Features
-        - text: "`qenv`"
-          href: articles/blueprint/qenv.html
+        - text: teal_data
```

Comment:
An idea:
as we already have "In-App Data" - should we change it to the "Input Data"?

Reply:
Not bad :+1: